settings/common.py

```diff
@@ -488,8 +488,16 @@ def get_release():
     'IS_RAVEN_INSTALLED': RAVEN_CONFIG['dsn'] != ''
 }
 
-# SOCIAL MEDIA CONFIGURATION
-# -----------------------------------------------------------------------------
+
+# SOCIAL MEDIA CONFIGURATIONS
+# --------------------------------------------------------------------
+
+LINKEDIN_AUTH = {
```

Comment:
I think you can move this, below `LIMIT_POSTS` and `MAX_POSTS_AT_ONCE`. Since these 2 variables are common to `SOCIAL_MEDIA_CONFIGURATION` and can be used by any other platform too.